src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4902,4 +4903,4 @@ namespace Akka.Util.Reflection
     {
         public static System.Collections.Generic.IEnumerable<object> GetArguments(this System.Linq.Expressions.NewExpression newExpression) { }
     }
-}
\ No newline at end of file
+}
```

Comment:
@Silv3rcircl3 It did help thanks ! would be more careful next time.

Just one more thing...windows-multinode-tests are failing now...can a new public api break it possibly ?

sorry about my noobish questions !